src/formatDate.js

```diff
@@ -50,7 +50,46 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const SEPARATOR_INDEX = 3;
+  const START_ITERATION = 0;
+  const END_ITERATION = 3;
+  const stringItems = date.split(fromFormat[SEPARATOR_INDEX]);
+  const newDate = [];
+
+  for (let i = START_ITERATION; i < END_ITERATION; i++) {
+    let toFormatItem = toFormat[i];
+
+    if (!fromFormat.includes(toFormatItem)) {
+      const SHORT_YEAR = 'YY';
+      const LONG_YEAR = 'YYYY';
+
+      if (toFormatItem === LONG_YEAR) {
+        toFormatItem = SHORT_YEAR;
+
+        const currentYear = stringItems[fromFormat.indexOf(SHORT_YEAR)];
+        const LAST_DIGITS = 30;
+        const CURRENT_CENTURY_YEAR = '20';
+        const PREVIOUS_CENTURY_YEAR = '19';
```

Comment:
it's better practice to declare static variables in top of the file
so, please move these top